ui/js/controllers/main.js

```diff
@@ -564,6 +572,16 @@ treeherderApp.controller('MainCtrl', [
         $scope.pinnedJobs = thPinboard.pinnedJobs;
         $scope.jobFilters = thJobFilters;
 
+        $scope.isShowDuplicateJobs = function() {
+            return $location.search().duplicate_jobs !== 'hidden';
```

Comment:
talked in IRC about reversing the logic here.  so 'visible' instead of 'hidden', I guess..